### PR TITLE
Changed the way EmbeddedCassandra finds the classpath

### PR DESCRIPTION
**Pull request by markncooper**
When running from Eclipse / ScalaTest java.class.path was often not setup.

### CODE REVIEW

File: spark-cassandra-connector-embedded/src/main/scala/com/datastax/spark/connector/embedded/EmbeddedCassandra.scala [hunk 1]
```diff
@@ -72,8 +72,13 @@ private[connector] class CassandraRunner(val configTemplate: String) extends Emb
       copyTextFileWithVariableSubstitution(input, output, properties)
     }
   }
+  private def makeJarPath() : String ={
```
Comment (helena) [inline spark-cassandra-connector-embedded/src/main/scala/com/datastax/spark/connector/embedded/EmbeddedCassandra.scala]: Don't add parens if there are no args and it's not fu(): Unit = {}

private def makeJarPath: String = {

File: spark-cassandra-connector-embedded/src/main/scala/com/datastax/spark/connector/embedded/EmbeddedCassandra.scala [hunk 2]
```diff
@@ -72,8 +72,13 @@ private[connector] class CassandraRunner(val configTemplate: String) extends Emb
...
+    var sysClassLoader : ClassLoader = ClassLoader.getSystemClassLoader();
```
Comment (helena) [inline spark-cassandra-connector-embedded/src/main/scala/com/datastax/spark/connector/embedded/EmbeddedCassandra.scala]: var? looks like val

this is not java, no semi-colon at the end of a statement ;)
```ClassLoader.getSystemClassLoader```

File: spark-cassandra-connector-embedded/src/main/scala/com/datastax/spark/connector/embedded/EmbeddedCassandra.scala [hunk 3]
```diff
@@ -72,8 +72,13 @@ private[connector] class CassandraRunner(val configTemplate: String) extends Emb
...
+    sysClassLoader.asInstanceOf[java.net.URLClassLoader].getURLs().
+                                map( e => e.getFile).mkString(":")
+  }  
 
-  private val classPath = System.getProperty("java.class.path")
```
Comment (helena) [inline spark-cassandra-connector-embedded/src/main/scala/com/datastax/spark/connector/embedded/EmbeddedCassandra.scala]: sys.props.get("java.class.path") 
I'd use the option

File: spark-cassandra-connector-embedded/src/main/scala/com/datastax/spark/connector/embedded/EmbeddedCassandra.scala [hunk 4]
```diff
@@ -72,8 +72,13 @@ private[connector] class CassandraRunner(val configTemplate: String) extends Emb
       copyTextFileWithVariableSubstitution(input, output, properties)
     }
   }
+  private def makeJarPath() : String ={
+    var sysClassLoader : ClassLoader = ClassLoader.getSystemClassLoader();
+    sysClassLoader.asInstanceOf[java.net.URLClassLoader].getURLs().
+                                map( e => e.getFile).mkString(":")
+  }
```
Comment (helena) [inline spark-cassandra-connector-embedded/src/main/scala/com/datastax/spark/connector/embedded/EmbeddedCassandra.scala]:
```private def jarPath: String = { 
    ClassLoader.getSystemClassLoader.asInstanceOf[java.net.URLClassLoader]
     .getURLs.map(_.getFile).mkString(":")
  }

**Comment by helena** [comment]
Added a few suggestions, have not tested this myself yet.

File: spark-cassandra-connector-embedded/src/main/scala/com/datastax/spark/connector/embedded/EmbeddedCassandra.scala (hunk 2 above)
Comment (pkolaczk) [inline spark-cassandra-connector-embedded/src/main/scala/com/datastax/spark/connector/embedded/EmbeddedCassandra.scala]: also, this should be named as a noun; e.g. jarPath

**Comment by pkolaczk** [comment]
@markncooper can you apply helena's suggestions and update the PR? I'd like to include that in 1.2.0-alpha 1. Thanks.

**Comment by markncooper** [comment]
Thanks for the comments! As is probably obvious, I'm pretty new to the world of Scala.

**Comment by markncooper** [comment]
Oops, the least I can do is make sure the code compiles. Sorry.

**Comment by jacek-lewandowski** [comment]
@markncooper can you squash your commits?

**Comment by jacek-lewandowski** [comment]
@markncooper any updates ?

**Comment by RussellSpitzer** [comment]
Is this still relevant? I'll close it out if no one responds in a bit

**Comment by markncooper** [comment]
I think it's not - close away!

**Pull request closed without merging**